common/test-utils/package.json

```diff
@@ -39,23 +39,23 @@
     "@testing-library/react-hooks": "^8.0.1",
     "@testing-library/user-event": "^14.4.3",
     "jest-axe": "^6.0.0",
-    "jest-fail-on-console": "^2.4.2",
-    "jest-matcher-utils": "^28.1.3",
-    "jest-environment-jsdom": "^28.1.3",
+    "jest-fail-on-console": "^3.0.1",
+    "jest-matcher-utils": "^29.0.1",
+    "jest-environment-jsdom": "^29.0.1",
     "jest-transform-stub": "^2.0.0",
-    "jest": "^28.1.3",
+    "jest": "^29.0.1",
     "identity-obj-proxy": "^3.0.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "ts-jest": "^28.0.7"
+    "resize-observer-polyfill": "^1.5.1",
```

Comment:
I think this is a devDependencies correct?

Reply:
This dependency needs to be installed and required when you install `@fuel-ui/test-utils` so, needs to be a dependency.